### PR TITLE
Update of contentcard: creator + institution

### PR DESCRIPTION
**Pull request by mirjamverloop**

### CODE REVIEW

File: components/generic/ContentCard.vue [hunk 1]
```diff
@@ -50,13 +79,40 @@
...
+    },
+    computed: {
+      linkToRecord () {
+        return this.localePath({ name: 'record-all', params: { pathMatch: this.recordId.replace(/^\/+/g, '') } });
+      },
+      backgroundImage () {
+        if (this.recordThumbnailUrl) {
+          return this.recordThumbnailUrl;
+        } else {
+          return (this.image && this.image.fields) ? this.image.fields.file.url : '';
```
Comment (rwd) [inline components/generic/ContentCard.vue]: I think that this component ought only need be aware of an image URL being passed to it, in an `imageUrl` prop. Provision of image URLs from different sources should be handled higher up the rendering hierarchy, perhaps even at plugin or page level.

File: components/browse/ContentCardSection.vue [hunk 2]
```diff
@@ -19,13 +19,16 @@
...
+        v-for="card in cards"
         :key="card.sys.id"
         :name="card.fields.name"
         :description="card.fields.description"
         :url="card.fields.url"
-        :image-url="card.fields.image.fields.file.url"
-        :image-title="card.fields.image.fields.title"
+        :image="card.fields.image"
+        :creator="card.fields.creator"
+        :institution="card.fields.provider"
```
Comment (rwd) [inline components/browse/ContentCardSection.vue]: Unless there is a good reason not to, reuse the name of the property from `card.fields`, i.e. `:provider`. Same for the following two attributes.

File: components/generic/ContentCard.vue (hunk 1 above)
Comment (mirjamverloop) [reply components/generic/ContentCard.vue]: I have moved it one level up to the content card section.

File: components/browse/ContentCardSection.vue (hunk 2 above)
Comment (mirjamverloop) [reply components/browse/ContentCardSection.vue]: I have renamed these.

**Pull request merged**